astro.config.mjs

```diff
@@ -30,10 +30,11 @@ const config = defineConfig({
       title: 'Contribute | freeCodeCamp.org',
       description: 'Contribute to freeCodeCamp.org',
       logo: {
-        src: './public/icons/icon-96x96.png',
+        light: './public/icons/secondary_logo.svg',
+        dark: './public/icons/primary_logo.svg',
         replacesTitle: true
       },
-      favicon: './public/favicon-32x32.png',
+      favicon: './public/favicon.ico',
```

Comment:
https://fix-header.contribute-5pf.pages.dev/public/favicon.ico isn't showing up. Is this a preview issue?

Reply:
Nope. I need to investigate.

Reply:
Ah, so the contents of "public" get hosted on "/"? It's showing up now, anyway.